package.json

```diff
@@ -24,26 +25,25 @@
     "@mapbox/sphericalmercator": "1.2.0",
     "@mapbox/vector-tile": "1.3.1",
     "advanced-pool": "0.3.3",
-    "canvas": "2.9.3",
```

Comment:
For dependencies management, we should use dependabot or renovate bot instead of manually updating the deps

Reply:
I agree. before there were a few dependencies that were stuck at old versions due to the old version of node, but if we raise the minimum node version that becomes a bit easier and we can automatically update a few things

Reply:
I removed unnecessary package updates from this pr. we can update them later with a more official method. there were a few devdeps relating to test i left and commander which I updated the syntax.